### PR TITLE
Gun attachments can now be fired with rclick

### PR DESCRIPTION
**Pull request by TiviPlus**
## About The Pull Request

Title

## Changelog
:cl:
add: Gun attachments can now be fired with rclick
/:cl:

### CODE REVIEW

**Comment by ilikesaying** [comment]
Alt Fire time

File: code/modules/projectiles/gun_system.dm [hunk 1]
```diff
@@ -1107,12 +1107,12 @@ and you're good to go.
...
 	if(attaching.flags_attach_features & ATTACH_PROJECTILE)
 		return //These are handled through regular Fire() for now.
-	RegisterSignal(src, list(COMSIG_ITEM_MIDDLECLICKON, COMSIG_ITEM_SHIFTCLICKON), .proc/do_fire_attachment) //For weapons with special projectiles not handled via Fire()
+	RegisterSignal(src, list(COMSIG_ITEM_MIDDLECLICKON, COMSIG_ITEM_SHIFTCLICKON, COMSIG_ITEM_RIGHTCLICKON), .proc/do_fire_attachment) //For weapons with special projectiles not handled via Fire()
 
 
 /obj/item/weapon/gun/proc/on_gun_attachment_detach(obj/item/attachable/attached_gun/detaching)
 	active_attachable = null
-	UnregisterSignal(src, list(COMSIG_ITEM_MIDDLECLICKON, COMSIG_ITEM_SHIFTCLICKON))
+	UnregisterSignal(src, list(COMSIG_ITEM_MIDDLECLICKON, COMSIG_ITEM_SHIFTCLICKON, COMSIG_ITEM_RIGHTCLICKON))
```
Comment (SplinterGP) [inline code/modules/projectiles/gun_system.dm:1115]: my god these fire on shift click ? i'd remove that

Comment (TiviPlus) [reply code/modules/projectiles/gun_system.dm:1115]: Is that a request or a suggestion?

Comment (SplinterGP) [reply code/modules/projectiles/gun_system.dm:1115]: suggestion

**Review by SplinterGP — APPROVED** [review]

**Pull request merged**